app/api/report.py

```diff
@@ -1057,3 +1057,126 @@ def junior_monthly_report():
 
     return jsonify(report_all)
 
+
+#Api for get juniors managers details.
+@bp.route('/get_managers', methods=['GET'])
+@jwt_required
+def get_managers():
+    current_user = get_current_user()
+    managers = current_user["managers"]
+    if not managers:
+        return jsonify(
+            {"msg": "currently you don't have any manager assigned"}),204
+    else:
+        return jsonify(managers)
+
+
+#api for 360_review
+@bp.route("/360_reviews", methods=["GET", "POST"])
+@jwt_required
+def reviews_360():

```

Comment:
where is this used?

Reply:
This route is using for sending juniors comments to managers using POST methods . and get methods for user can see his response.